ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -397,6 +397,12 @@ module.exports = function CesiumMap(
         destination: Cesium.Rectangle.fromDegrees(west, south, east, north),
       })
     },
+    getBoundingBox: function() {
+      const viewRectangle = map.scene.camera.computeViewRectangle()
+      return _.mapObject(viewRectangle, function(val, key) {
```

Comment:
✏️ I would suggest doing `Object.values(viewRectangle).map(val => Cesium.Math.toDegrees(val))` as a way to not depend on another library :)

Reply:
Doesn't `Object.values` return an array of the values? I need to return an object with the same field names, but with each value converted from radians to degrees.

Reply:
However, I can clean it up a bit by using an arrow function.

Reply:
Yes, this was only meant as an example 😄 You could do `Object.entries`...and then
```
.map(entry => ({entry.key: Cesium.Math.toDegrees(entry.value)})
```
or some such

Reply:
In this case, I'm going to leave the code the way it is. The library I'm using is already widely used in this file and throughout the project.